unidist/test/test_async_actor.py

```diff
@@ -88,3 +88,46 @@ def test_address_space(is_use_options):
 
     assert_equal(object_ref0, 0)
     assert_equal(object_ref1, 1)
+
+
+@pytest.mark.skipif(
+    Backend.get() == BackendName.MP,
+    reason="Proper serialization/deserialization is not implemented yet for multiprocessing",
+)
+def test_global_capture():
+    actor = TestAsyncActor.remote(0)
+
+    @unidist.remote
+    def foo():
+        object_ref = actor.task.remote(5)
```

Comment:
```suggestion
        object_ref = actor.get_accumulator.remote()
```
Let's be in line with tests for TestActor.